src/main/groovy/com/github/benmanes/gradle/versions/reporter/result/Dependency.groovy

```diff
@@ -15,4 +15,5 @@ class Dependency {
   String name
   String version
   String projectUrl
+  String userReason
```

Comment:
I used "UserReason" because "Reason" was already being used for unresolved dependencies.